zvariant/src/array.rs

```diff
@@ -67,7 +67,7 @@ impl<'a> Array<'a> {
     /// Get the value at the given index.
     pub fn get<V>(&'a self, idx: usize) -> Result<Option<V>>
     where
-        V: ?Sized + TryFrom<&'a Value<'a>>,
+        V: TryFrom<&'a Value<'a>>,
```

Comment:
> warning: `?Sized` bound is ignored because of a `Sized` requirement

Where is the `Sized` requirement coming from? Please copy&past the full warning/error so it's clear.

Protip: best to also specify the syntax as `rust` even if the error/warning isn't exactly all Rust. The syntax highlighting still makes the message much nicer/easier to read.

Reply:
`TryFrom` has the bound `Sized`, update commit comment with clippy warning